packages/flutter/lib/src/rendering/custom_layout.dart

```diff
@@ -15,21 +16,30 @@ class MultiChildLayoutParentData extends ContainerBoxParentDataMixin<RenderBox>
 }
 
 /// A delegate that controls the layout of multiple children.
+///
+/// Used with [MultiChildCustomLayout], the widget for the
+/// [RenderCustomMultiChildLayoutBox] render object.
+///
+/// Subclass this object and override some or all of the methods
```

Comment:
Subclass this object sounds a little obtuse. Maybe just say "Subclasses must override some or ..."